core/src/main/java/hudson/model/UpdateSite.java

```diff
@@ -1361,7 +1361,7 @@ public void install() {
             for (Plugin dep : getNeededDependencies()) {
                 UpdateCenter.InstallationJob job = uc.getJob(dep);
                 if (job == null || job.status instanceof UpdateCenter.DownloadJob.Failure) {
-                    LOGGER.log(Level.INFO, "Adding dependent install of " + dep.name + " for plugin " + name);
+                    LOGGER.log(Level.FINEST, "Adding dependent install of " + dep.name + " for plugin " + name);
```

Comment:
This is a useful message IMO, especially since the UI does a poor job of communicating dependency installations.